### ISSUE TITLE
Feature Request: Show duplicate passwords already used at other accounts

### ISSUE THREAD

**Issue by stefan1983** [keepassxreboot/keepassxc#2203]
It would be nice if it could be shown, when a password is already used for other accounts. This educates everyone to use passwords only ONCE and not for multiple accounts.

**Comment by droidmonkey** [comment]
This is a duplicate of #551
